samples-logs-back-end/cronjob/sync.py

```diff
@@ -8,18 +8,23 @@
 
 def main():
     records = collect_table_data()
-    create_tmp_collection(records)
-    download_files_process = subprocess.run(
-        "wget http://45.86.170.46/coronavirus_sequence.tsv", shell=True,
-        capture_output=True)
-    update_tmp_collection()
+    create_tmp_collections(records)
+    update_tmp_phylo_collection()
+    update_tmp_lineage_collection()
+    copy_to_prod_collections()
+
+
+def create_tmp_collections(records):
+    DB.phylo_tmp.insert_many(records)
+    DB.lineage_tmp.insert_many(records)
 
 
 def collect_table_data():
     """
     This function will collect all records from
     https://www.covid19dataportal.org/sequences?db=embl-covid19 table
-    :return: list of records to be added to DB
+    :return: list of records to be added to DB in format eg.
+    [{'acc': 'MT947591', 'id': 'MT947591', 'source': 'embl-covid19'}, ...]
```

Comment:
Can you also update this, on data portal first two records always are: MN908947, LR991698 and they don't come from EBI-search - https://www.covid19dataportal.org/sequences?db=embl-covid19

Reply:
Re: they don't come from EBI-search: Then is the url on lines 25, 31 and 40 correct? (https://www.ebi.ac.uk/ebisearch/ws/rest/embl-covid19/ ...)

Reply:
So for a first page request we just always need to add these two records on the top